src/App.tsx

```diff
@@ -15,48 +18,108 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  ALPHABETICALLY = 'Sort alphabetically',
+  BY_LENGTH = 'Sort by length',
+  NONE = '',
```

Comment:
I'm not sure that uppercase is a good practice for naming enum. Also it is easier to understand matching names
```suggestion
  alphabet = 'alphabet',
  length = 'length',
  none = '',
```